pkg/capabilities/consensus/ocr3/factory.go

```diff
@@ -32,6 +32,8 @@ type factoryService struct {
 const (
 	defaultRequestExpiry time.Duration = 1 * time.Hour
 	defaultBatchSize                   = 1000
+	maxPhaseOutputBytes                = 100000
+	maxReportCount                     = 20
```

Comment:
Where are these numbers coming from? If they are arbitrary, could this be called out in a comment?

Reply:
hmm do you want a comment that says "arbitrary"? How about I rename to "defaultMax.."? The existing two ones above are also arbitrary.

Reply:
> hmm do you want a comment that says "arbitrary"?

Yes, if that's the case. When someone looks at this later, we don't want them to spend hours trying to understand why this was set to these specific numbers 😅